Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/impl/apmlog/LogFilterControllerWithAPMLog.java

```diff
@@ -248,6 +261,23 @@ private RuleValue findRuleValueByOpeType(OperationType operationType, Set<RuleVa
                 .orElse(null);
     }
 
+    public static JSONObject getDefaultCaseTabAttributePayload(String label) {
+        JSONObject payload = new JSONObject();
+        payload.put("type", "CaseTabAttribute");
+        payload.put("key", convertAttributeKey(label));
+        payload.put("data", "");
+        return payload;
+    }
+
+    private static String convertAttributeKey(String label) {
+        switch (label) {
+            case "Resources": return "org:resource";
+            case "Activities": return "concept:name";
+            case "Groups": return "org:group";
+            default: return label;
+        }
+    }
+
```

Comment:
Test with "Production_Data" log, open it in PD, select "Qty completed" or "Qty rejected" in the perspective list, note that the Activity heading in the stat box changes to "Qty completed", click on it, error is shown below.

![Uploading image.png…]()